corehq/apps/locations/views.py

```diff
@@ -117,6 +119,9 @@ class LocationTypesView(BaseLocationView):
     template_name = 'locations/location_types.html'
 
     @method_decorator(can_edit_location_types)
+    @use_bootstrap3
+    @use_knockout_js
```

Comment:
this decorator was removed...please pull the latest master so that this gets updated, thanks!